pkg/capabilities/consensus/ocr3/factory.go

```diff
@@ -1,28 +1,34 @@
 package ocr3
 
 import (
+	"context"
+
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
 )
 
 type factory struct {
-	store *store
-	*capability
-	batchSize int
-	lggr      logger.Logger
+	store      *store
+	capability *capability
+	batchSize  int
+	lggr       logger.Logger
+
+	services.StateMachine
 }
 
 const (
 	defaultMaxPhaseOutputBytes = 100000
 	defaultMaxReportCount      = 20
 )
 
-func newFactory(s *store, batchSize int, lggr logger.Logger) (*factory, error) {
+func newFactory(s *store, c *capability, batchSize int, lggr logger.Logger) (*factory, error) {
```

Comment:
@bolekk Some context here: previously we were embedding capability (which is a service), but I evidently forgot to actually plug it in. This made factory implement the service interface even though I had removed that in my previous PR.

Unfortunately, factory does need to be a service -- that's a mandate of the ReportingPluginFactory type, so I've stubbed out the relevant functions and removed the embedding of the capability. The capability is instantiated elsewhere and I don't want to conflate calling start on the factory (which rightfully should noop) with calling start on the capability.

Reply:
It's part of the LOOPP reporting plugin definition (chainlink-common/pkg/types/reporting_plugin_service.go) and is used by the LOOPP framework.